src/js/plugins/inheritance.js

```diff
@@ -1634,6 +1634,35 @@
 
     init: function () {
       var self = this;
+      
+      // Watch the body for attribute changes indicating a card has loaded.
+      // This function uses a MutationObserver to check if the body has any
+      // attribute change (which loading a card will cause). If a card loads,
+      // we immediately call updateCardView which mounts the "Parent," 
+      // and "Child," buttons.
+      (function () {
+        // Target element that we will observe
+        const target = document.body;
+    
+        // config object
+        const config = {
+          attributes: true,
+        };
+    
+        // Subscriber function
+        function subscriber(mutations) {
+          mutations.forEach((mutation) => {
+            // Handle mutations here by calling updateCardView.
```

Comment:
Does the comment tell us anything the code doesn't here? I prefer that we make the code tell the story of "what" clearly, and generally keep comments for telling us "why". (Fewer things to maintain that way!)

Reply:
Agreed, I'll be going back to clean up a few comments.